Cargo.toml

```diff
@@ -1,20 +1,73 @@
 [package]
-
-name         = "proteus"
-version      = "1.0.4"
+name         = "proteus-wasm"
```

Comment:
Is this just gonna be used in wasm builds?

Reply:
No... This a version that works with wasm and needed a new name to not conflict with the old version when both are present in the dependency tree.